worlds/alttp/Options.py

```diff
@@ -1,11 +1,18 @@
 import typing
 
 from BaseClasses import MultiWorld
-from Options import Choice, DeathLink, DefaultOnToggle, ItemsAccessibility, Option, PlandoBosses, Range, \
-    StartInventoryPool, Toggle
-
-
-class Logic(Choice):
+from Options import Choice, Range, Option, Toggle, DefaultOnToggle, DeathLink, StartInventoryPool, PlandoBosses,\
+    FreeText, ItemsAccessibility
```

Comment:
Is there a reason for changing the order of import names here?
Some people like alphabetical order (like they were before this change).

Reply:
I see now that it was changed from non-alphabetical to alphabetical, then reverted. (That's more understandable. I'm not trying to enforce alphabetical.)